.github/workflows/phpunit-ci-coverage.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - main
+      - test
```

Comment:
Test branch should be removed
```suggestion
```

Reply:
Thanks for the catch, missed this from my own testing